controller/doc.js

```diff
@@ -23,7 +23,7 @@ function setup( backend ){
       if( err ){ return next( err ); }
 
       // convert docs to geojson
-      var geojson = geojsonify( docs );
+      var geojson = geojsonify( docs, req.clean.details );
```

Comment:
minor comment, if the second arg was an object then it could be more easily extended to allow a range of flags to be passed. as it is, if you want to add another flag-type-thing you'll either need to refactor this or add another new arg.